android/expoview/src/main/java/versioned/host/exp/exponent/modules/universal/notifications/ScopedExpoNotificationCategoriesModule.java

```diff
@@ -0,0 +1,70 @@
+package versioned.host.exp.exponent.modules.universal.notifications;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.os.ResultReceiver;
+
+import org.unimodules.core.Promise;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+
+import expo.modules.notifications.notifications.NotificationSerializer;
+import expo.modules.notifications.notifications.categories.ExpoNotificationCategoriesModule;
+import expo.modules.notifications.notifications.model.NotificationCategory;
+import expo.modules.notifications.notifications.service.BaseNotificationsService;
+import host.exp.exponent.kernel.ExperienceId;
+
+public class ScopedExpoNotificationCategoriesModule extends ExpoNotificationCategoriesModule {
+  private String mExperienceIdString;
+
+  public ScopedExpoNotificationCategoriesModule(Context context, ExperienceId experienceId) {
+    super(context);
+    if (experienceId == null) {
```

Comment:
I'm not sure, I don't see how. But other scoped module constructors null-check the experience ID (`experienceId` passed to `ScopedExpoNotificationPresentationModule` is later null checked in [`ScopedNotificationRequest`](https://github.com/expo/expo/blob/master/android/expoview/src/main/java/host/exp/exponent/notifications/ScopedNotificationRequest.java#L18-L25)) so it seemed best to follow suit rather than risk a possible NullPointerException